InterfaceStubGenerator.Shim/ModuleInitializer.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace Refit.Generator
+{
+    internal static class ModuleInitializer
+    {
+        internal static void Initialize()
```

Comment:
You need to apply the module initialize attribute.

Reply:
➡️ This is fixed